src/mds/MDSRank.cc

```diff
@@ -36,11 +36,244 @@
 
 #include "MDSRank.h"
 
+#include <boost/scope_exit.hpp>
```

Comment:
Let's use `src/include/scope_guard.h` instead.

Reply:
this will be removed with the latest update...